.gitignore

```diff
@@ -19,3 +19,4 @@ demo/config/master.key
 primer_view_components*.gem
 .DS_Store
 app/components/**/*.js
+app/components/**/*.d.ts
```

Comment:
if we don't commit them, how will publish work?
Will it rely on the local files from the person that ran `script/publish`?

Reply:
Yes, the local type declaration files will be published, as the local JS files are currently. 👍